tests/bud.bats

```diff
@@ -333,7 +333,7 @@ _EOF
 }
 
 @test "bud build with heredoc content" {
-  _prefetch fedora
+  _prefetch quay.io/fedora/python-311
```

Comment:
please only ever use images from quay.io/libpod/...
Second to we actually have to pull a large python image for this? I don't see the relevance of python specific code in herdoc. Can just as well test a simple shell script IMO.

Reply:
There's probably some value in checking that the right thing happens when the designated consumer of the heredoc is something other than the shell.  I don't think anything the quay.io/libpod organization includes a python interpreter, but if there is one, I wouldn't mind using it instead.